### ISSUE TITLE
Standardize MFEs to reduce total complexity

### ISSUE THREAD

**Issue by jmakowski1123** [openedx/axim-engineering#432]
### Problem

Instead of one complex monolith, we now have several less-complex projects in addition to the big one we started with.  Plus, while less complex individually, MFEs are collectively harder to work with.

### Product/Platform Value

Reducing the cost of onboarding is also an important aspect of fostering adoption: the easier it is for providers to train their developers, the more use-cases the platform can be used for.

### Acceptance Criteria

Standardize MFEs as much as possible, so that a good part of developer and deployment complexities can be learned once and reused across repositories, all while avoiding freezing the technology in time (which is what happened with edx-platform).

### Contingencies

This should be scoped in parallel with the epic to finish the MFE conversion process.

**Comment by arbrandes** [comment]
Closing in favor of https://github.com/openedx/axim-engineering/issues/437.
